### PR TITLE
File Structure and Design Changes

### PR DESCRIPTION
**Pull request by Jbrooks00**
Attempting to merge the most recent code base for sprint 2 code review practice.

My development
* Inbox Development: Implemented a userProfile interface
* Restructured our files for sprint 2. I added pages folder separate from main components. 
* Implemented a profile management page that can upload a photo to Firebase

Other Contributors

Jim - Fixed bugs within the login page
Alexandru - Continue development for footer, homepage,
Anthony - Continue development on Nav-bar and my-account

### CODE REVIEW

**Comment by Jbrooks00** [comment]
Not sure why this is causing conflict... I had thought I managed them before commit. If someone could take a look.

File: .gitignore [hunk 1]
```diff
@@ -40,11 +40,7 @@ testem.log
 # System files
 .DS_Store
 Thumbs.db
-<<<<<<< HEAD
```
Comment (Jbrooks00) [inline .gitignore:43]: Merge conflict resolution...

File: src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+import { MaintenancepageComponent } from './public/Pages/maintenancepage/maintenancepage.component';
+import { ReportissuepageComponent } from './public/Pages/report-issue-page/reportissuepage.component';
+import { SearchrepairspageComponent } from './public/Pages/searchrepairspage/searchrepairspage.component';
+import { SignupComponent } from './public/Pages/signup/signup.component';
+import { MySettingsComponent } from './private/pages/mySettings/my-settings.component';
+import { MyInboxComponent } from './private/pages/myInbox/my-inbox.component';
+
+import { canActivate, redirectUnauthorizedTo, redirectLoggedInTo } from '@angular/fire/auth-guard';
+import { ProfileComponent } from './private/pages/profile/profile.component';
+
```
Comment (Jbrooks00) [inline src/app/app-routing.module.ts:19]: Implemented basic navbar component pages

File: src/app/app.component.html [hunk 3]
```diff
@@ -3,6 +3,6 @@
 <app-navbar>
 </app-navbar> <!--Static Navbar - Anthony -->
 
-<div class="content">
+<div>
```
Comment (Jbrooks00) [inline src/app/app.component.html:6]: Bug?

File: src/app/app.module.ts [hunk 4]
```diff
@@ -1,18 +1,18 @@
+//Core imports
```
Comment (Jbrooks00) [inline src/app/app.module.ts:1]: I  re worked the import structure and actually imported chat service dependencies.

File: src/app/core/auth/auth.service.ts [hunk 5]
```diff
@@ -23,4 +25,16 @@ export class AuthenticationService {
...
+
+  updateProfileData(profileData: Partial<UserInfo>): Observable<any> {
+    const user = this.auth.currentUser;
+     return of(user).pipe(
+       concatMap((user) => {
+         if (!user) throw new Error('Not authenticated');
+
+         return updateProfile(user, profileData);
+       })
+     );
```
Comment (Jbrooks00) [inline src/app/core/auth/auth.service.ts:37]: This method requires partial information of a user based off of the profileData interface to update the database.

File: src/app/private/pages/myAccount/my-account.component.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
+<html>
```
Comment (Jbrooks00) [inline src/app/private/pages/myAccount/my-account.component.html:1]: A non-functional template for presentation

**Review by Jbrooks00 — COMMENTED** [review]
Overview, a lot of a deleted files, use file filter for .ts files : ]

File: src/app/public/components/loginpage/loginpage.component.html [hunk 7]
```diff
@@ -1,28 +0,0 @@
-<div class="form-container mat-elevation-z5">
```
Comment (Anth0nyQ) [inline src/app/public/components/loginpage/loginpage.component.html:1]: It looks like the merge conflict is coming from the fact that the file is empty? This is weird. I'm looking into it.

Comment (Anth0nyQ) [reply src/app/public/components/loginpage/loginpage.component.html:1]: We need to delete this file from the commit. It's causing issues merging and I am unsure of how to resolve it without manually deleting it, but it's one of Jim's files. Whenever you get a chance can you check this out?

Comment (Anth0nyQ) [reply src/app/public/components/loginpage/loginpage.component.html:1]: I also have no idea where the matching .ts file is for this file, so I'm majorly lost on that issue with the merge conflict.

**Review by Anth0nyQ — APPROVED** [review]
We have to figure out the issue with GitHub not recognizing that the pages were deleted/moved to a different folder and have to manually go in and delete them whenever Jim gets a minute to do so.

**Review by JimDUc — APPROVED** [review]
Pull request reviewed, no additional comments needed.

**Pull request closed without merging**